tests/test-security.php

```diff
@@ -116,6 +119,40 @@ public function test__lost_password_limit() {
 
 	}
 
+	public function test__login_system_multicall_rate_limit() {
+		add_filter( 'pre_option_enable_xmlrpc', '__return_true' );
+		$myxmlrpcserver = new wp_xmlrpc_server();
+
+		$method = array(
+			'methodName' => 'wp.getUsersBlogs',
+			'params'     => array(
+				0,
+				'admin20',
+				'password',
+			),
+		);
+
+		$method_calls = array();
+
+		$limit_threshold = 50;
+		$last_threshold_index = $limit_threshold - 1;
+
+		for ( $i = 1; $i <= $limit_threshold + 2; $i++ ) {
+			array_push( $method_calls, $method );
+		}
+
+		$myxmlrpcserver->callbacks = $myxmlrpcserver->methods;
+
+		$result = $myxmlrpcserver->multiCall( $method_calls );
```

Comment:
```suggestion
		$this->_error_level = error_reporting();
		error_reporting( $this->_error_level & ~E_WARNING );
		$result = $myxmlrpcserver->multiCall( $method_calls );
		error_reporting( $this->_error_level );
```
Suppress the `headers already sent` error for this test.